python/ray/tune/suggest/nevergrad.py

```diff
@@ -202,7 +223,15 @@ def suggest(self, trial_id: str) -> Optional[Dict]:
         if self.max_concurrent:
             if len(self._live_trial_mapping) >= self.max_concurrent:
                 return None
+
+        if len(self._points_to_evaluate) > 0:
+            point_to_evaluate = self._points_to_evaluate.pop(0)
+            for key in self._space.value:
+                if isinstance(self._space[key], ng.p.Choice):
+                    point_to_evaluate[key] = self._space[key].choices.value[point_to_evaluate[key]]
```

Comment:
Question: Couldn't we pass e.g. `activation: "relu"` in the `points_to_evaluate` dict directly and omit this lookup?

Reply:
That's for sure. I did like this in the beginning, but I change it to be compatible with points_to_evaluate from HyperOptSearch.

Reply:
Let's do it with strings then (e.g. `activation: "relu"`) and we'll change the hyperoptsearch implementation in a different PR.

Reply:
done